packages/celotool/src/lib/ethstats.ts

```diff
@@ -29,13 +30,26 @@ export async function upgradeHelmChart(celoEnv: string) {
 function helmParameters() {
   return [
     `--set domain.name=${fetchEnv(envVar.CLUSTER_DOMAIN_NAME)}`,
-    `--set ethstats.createSecret=${isVmBased()}`,
-    `--set ethstats.webSocketSecret="${fetchEnv(envVar.ETHSTATS_WEBSOCKETSECRET)}"`,
     `--set ethstats.image.repository=${fetchEnv(envVar.ETHSTATS_DOCKER_IMAGE_REPOSITORY)}`,
     `--set ethstats.image.tag=${fetchEnv(envVar.ETHSTATS_DOCKER_IMAGE_TAG)}`,
+    `--set ethstats.trusted_enodes='{${generateAuthorizedAddresses()}}'`,
+    `--set ethstats.banned_enodes='{${fetchEnv(envVar.ETHSTATS_BANNED_ENODES)}}'`,
   ]
 }
 
 function releaseName(celoEnv: string) {
   return `${celoEnv}-ethstats`
 }
+
+function generateAuthorizedAddresses() {
+  // TODO: Add the Proxy eth addresses when available
+  const mnemonic = fetchEnv(envVar.MNEMONIC)
+  const publicKeys = []
+  const txNodes = parseInt(fetchEnv(envVar.TX_NODES), 0)
+  const validatorNodes = parseInt(fetchEnv(envVar.VALIDATORS), 0)
+  publicKeys.push(getAddressesFor(AccountType.TX_NODE, mnemonic, txNodes))
+  publicKeys.push(getAddressesFor(AccountType.VALIDATOR, mnemonic, validatorNodes))
+
+  publicKeys.push(fetchEnv(envVar.ETHSTATS_TRUSTED_ENODES).split(','))
+  return publicKeys.reduce((accumulator, value) => accumulator.concat(value), [])
+}
```

Comment:
@tkporter I adapted to the latest changes Dimi is finishing for using etherbase accounts. Should be ready soon (need to update the commits here). Meanwhile do you think this way is ok for getting the new addresses? It is concatenated the addresses from `getAddressesFor` with the ones from the env `ETHSTATS_TRUSTED_ENODES` (this one for external nodes submitting to celostats).